lib/esc.js

```diff
@@ -209,34 +207,22 @@ function ESC(opts) {
         return state.value;
       }
     },
-    history: {
-      get: function() {
-        return state.history.slice(-5);
-      }
-    },
     last: {
       get: function() {
-        return state.history[state.history.length - 1] || {
-          last: null
-        };
+        return state.last;
       }
     }
   }));
 
   this.initialize(opts);
 
   if (this.deviceName !== "FORWARD") {
-    if (Number.isNaN(+this.neutral)) {
-      throw new Error("Directional speed controllers require a neutral point from 0-100 (number)");
+    if (this.neutral === this.pwmRange[0]) {
+      throw new Error("Directional speed controllers require a non-zero neutral point");
     }
-
-    this.startAt = this.neutral;
   }
 
-  // Match either null or undefined, but not 0
-  if (this.startAt !== null && this.startAt !== undefined) {
-    this.speed(this.startAt);
-  }
+  this.throttle(this.neutral);
```

Comment:
This is a safety precaution. No program should ever just "start" an ESC without explicit instruction. This is the safest thing I can think of.